app/views/assignment_mailer/new_assignment.text.erb

```diff
@@ -1,9 +1,13 @@
 Hello, <%= @assignment.assigned_to.full_name %>,
 ===============================================
 
-You have been assigned a new action item by <%= @assignment.assigned_by.full_name %>.
+You have been assigned a new action item by <%= @assignment.assigned_by.full_name %>. A summary of your action item can be found below.
 
-To view your action item, log in using this link: <%= @url %>
+Assignment: <%= @action_item.title %>
+Due Date: <%= @due_date %>
+<%= @action_item.description %>
+
+To view your action item, sign into your Unloop account <%= link_to 'here', @assignment %>.
 
 Best,
 Unloop Staff
```

Comment:
Nitpick but try to keep the emails the same text

Reply:
oh got it i'll fix that thanks for the catch :)